devices/connecte.js

```diff
@@ -0,0 +1,44 @@
+const exposes = require('../lib/exposes');
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const tuya = require('../lib/tuya');
+const ea = exposes.access;
+
+module.exports = [
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_4hbx5cvx'}],
+        model: '4500994',
+        vendor: 'Connecte',
+        description: 'Smart termostat',
+        fromZigbee: [
+            fz.connecte_thermostat,
+        ],
+        toZigbee: [
+        ],
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            // Do a "magic" read on the basic cluster to trigger the termostat start reporting.
+            await endpoint.read('genBasic', ['manufacturerName', 'zclVersion', 'appVersion', 'modelId', 'powerSource', 0xfffe]);
+        },
+        exposes: [
+            exposes.binary('state', ea.STATE, true, false).withDescription('State of thermostat'),
+            exposes.numeric('system_mode', ea.STATE),
```

Comment:
`system_mode` is already included in the `exposes.climate()`, and also it is not a numerical value (`exposes.numeric`)

Reply:
Yes, true. Thanks, will clean up.